src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -211,6 +208,11 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
                 throw new InvalidEventTypeException("\"metadata\" property is reserved");
             }
 
+            if (eventType.getCompatibilityMode() == CompatibilityMode.DEPRECATED) {
+                throw new InvalidEventTypeException(
+                        "\"compatibility_mode\" should be either \"compatible\" or \"none\"");
```

Comment:
this is done because only old schemas will be "deprecated", and the new ones should never become "deprecated", right?